decidim-core/app/views/decidim/participatory_processes/_hero_process.html.erb

```diff
@@ -0,0 +1,27 @@
+<article class="card card--full card--process">
+  <div class="row collapse card--process__row">
+    <div class="columns mediumlarge-8 large-6 card--process__column">
+      <div class="card__content">
+        <%= link_to participatory_process_path(hero_process), class: "card__link" do %>
+          <h2 class="card__title"><%= translated_attribute hero_process.title %></h2>
+        <% end %>
+        <%= truncate(translated_attribute(hero_process.description).html_safe, length: 630, separator: ' ') %>
```

Comment:
I'd rather specify the `html_Safe` part instead of relying in the `<%==` part, as it's really similar to the non-safe `<%=` code and it can be difficult to differentiate